hack/image-mirroring/supplemental_ci_images_mirror_gen.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+import glob
+import shutil
+from ruamel.yaml import YAML
+
+APPCI_REGISTRY = "registry.ci.openshift.org"
+MAPPING_FILE_PREFIX = "mapping_"
+DIR_PATH = "core-services/image-mirroring"
+PERIODICS_FILE = "ci-operator/jobs/infra-image-mirroring-multi-arch.yaml"
+ARCHITECTURES = {
+    "arm64": "linux/arm64",
```

Comment:
Architecture listed here is only arm64, should the file name reflect that's only for aarch64 or alt archs?

Reply:
Once we have more clusters that support different architectures, we should include them here as well.